kinescope/translations/README.txt

```diff
@@ -0,0 +1,4 @@
+Use this translations directory to provide internationalized strings for your XBlock project.
```

Comment:
I did use `xblock-sdk` for this since the official [guide](https://edx.readthedocs.io/projects/xblock-tutorial/en/latest/getting_started/index.html) also uses this.

Having now looked at the code generated by the cookiecutter, it does seem more modern and in-line with other repos.

Is it ok to switch to the cookiecutter template in a follow-up PR ?

Reply:
@kaustavb12,
> Is it ok to switch to the cookiecutter template in a follow-up PR ?

Sure.